edge-agent/test/Microsoft.Azure.Devices.Edge.Agent.Kubernetes.IntegrationTest/cluster/KindClusterManager.cs

```diff
@@ -0,0 +1,48 @@
+// Copyright (c) Microsoft. All rights reserved.
+namespace Microsoft.Azure.Devices.Edge.Agent.Kubernetes.IntegrationTest.Cluster
+{
+    using System;
+    using System.Threading.Tasks;
+    using CliWrap;
+    using k8s;
+
+    public class KindClusterManager : IKubernetesClusterManager
+    {
+        readonly string name;
+
+        public KindClusterManager(string name)
+        {
+            this.name = name;
```

Comment:
is it required?

Reply:
Yeah. In order to run some tests in parallel, you may want to create separate clusters with different names.